src/main/scala/com/dwolla/cloudflare/domain/model/DnsRecord.scala

```diff
@@ -1,4 +1,4 @@
-package com.dwolla.cloudflare.model
+package com.dwolla.cloudflare.domain.model
```

Comment:
Ok. I guess if these classes are only used inside the library, then it won't impact the original project and makes sense to change. If we'll need to go back and update a bunch of imports in the original project, then I probably would have left it alone. Just my preference, though—I'm ok with leaving it now that the change has been made.